### DISCUSSION TITLE
Preview

### DISCUSSION THREAD

**Discussion by samchouse** [eclipse-theia/theia#7750] [General] [answered]
How do I open the preview pane (web browser version)?

**Comment by vince-fugnitto** [comment, marked as answer]
I answered the same question on [spectrum](https://spectrum.chat/theia/dev/how-do-i-open-the-preview-panel~2c73df6f-caa5-4dc4-90d7-6d7a88b7c5bf).
It is possible to open `previews` of resources which can be previewed such as images, html, and markdown, using the methods previously listed.
